src/http/representation/RepresentationPreferences.ts

```diff
@@ -7,7 +7,8 @@
   * where 0.001 is the least preferred and 1 is the most preferred; a
   * value of 0 means "not acceptable"."
   */
-export type ValuePreferences = Record<string, number>;
+// eslint-disable-next-line @typescript-eslint/consistent-indexed-object-style
```

Comment:
This should have a comment explaining why this is needed. Best would be to first open an issue in the Components.js explaining the issue so we can link to it in a comment here.

Reply:
Added a reference to: https://github.com/LinkedSoftwareDependencies/Components-Generator.js/issues/103